src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/ResourceExtensions.cs

```diff
@@ -17,7 +17,7 @@ public static OtlpResource.Resource ToOtlpResource(this Resource resource)
 
         foreach (KeyValuePair<string, object> attribute in resource.Attributes)
         {
-            OtlpTagWriter.Instance.TryWriteTag(ref processResourceAttributes, attribute);
+            OtlpTagWriter.Instance.TryWriteTag(ref processResourceAttributes, attribute.Key, attribute.Value);
```

Comment:
Is this related to nullable analysis? I don't have an issue with the change, just don't understand why it would be needed 😄

Reply:
TryWriteTag is accepting `KeyValuePair<string, object?>`. Rosources returing `KeyValuePair<string, object>` (objest is non-nullable). The cheapest way I have found, without repacking object was to create overload with two separate parameters - key and value. Happy to change if you see better option for fixes.